File: lat2/python/timsepakbola.py
class DaftarTimSepakbola():

	#private members 
	__namaTim = "";
	__asalNegara = "";
	__tahunTim = 0;
	__namaPemain = "";
	__nomorPemain = 0;
	__posisi = "";

	def __init__(self, namaTim = "", asalNegara = "", tahunTim = 0, namaPemain = "", nomorPemain = 0, posisi = ""):
		self.__namaTim = namaTim
		self.__asalNegara = asalNegara
		self.__tahunTim = tahunTim
		self.__namaPemain = namaPemain
		self.__nomorPemain = nomorPemain
		self.__posisi = posisi

	def setNamaTim(self, namaTim):
		self.__namaTim = namaTim
	def getNamaTim(self):
		return self.__namaTim

	def setAsalNegara(self, asalNegara):
		self.__asalNegara = asalNegara
	def getAsalNegara(self):
		return self.__asalNegara

	def setTahunTim(self, tahunTim):
		self.__tahunTim = tahunTim
	def getTahunTim(self):
		return self.__tahunTim

	def setNamaPemain(self, namaPemain):
		self.__namaPemain = namaPemain
	def getNamaPemain(self):
		return self.__namaPemain

	def setNomorPemain(self, nomorPemain):
		self.__nomorPemain = nomorPemain
	def getNomorPemain(self):
		return self.__nomorPemain

	def setPosisi(self, posisi):
		self.__posisi = posisi
	def getPosisi(self):
		return self.__posisi


	

File: lat2/php/timsepakbola.php
<?php

// membuat kelas untuk daftar tim sepakbola 
class DaftarTimSepakbola{

	private $namaTim = "";
	private $asalNegara = "";
	private $tahunTim = 0;
	private $namaPemain = "";
	private $nomorPemain = 0;
	private $posisi = "";


	public function __construct($namaTim= "", $asalNegara = "", $tahunTim = 0, $namaPemain = "", $nomorPemain = 0, $posisi = ""){
		$this->namaTim = $namaTim;
		$this->asalNegara = $asalNegara;
		$this->tahunTim = $tahunTim;
		$this->namaPemain = $namaPemain;
		$this->nomorPemain = $nomorPemain;
		$this->posisi = $posisi;
	}

	// set dan get dari private

	public function setNamaTim($namaTim){
		$this->namaTim = $namaTim;
	}
	public function setAsalNegara($asalNegara){
		$this->asalNegara = $asalNegara;
	}
	public function setTahunTim($tahunTim){
		$this->tahunTim = $tahunTim;
	}
	public function setNamaPemain($namaPemain){
		$this->namaPemain = $namaPemain;
	}
	public function setNomorPemain($nomorPemain){
		$this->nomorPemain = $nomorPemain;
	}
	public function setPosisi($posisi){
		$this->posisi = $posisi;
	}

	public function getNamaTim(){
		return $this->namaTim;
	}
	public function getAsalNegara(){
		return $this->asalNegara;
	}
	public function getTahunTim(){
		return $this->tahunTim;
	}
	public function getNamaPemain(){
		return $this->namaPemain;
	}
	public function getNomorPemain(){
		return $this->nomorPemain;
	}
	public function getPosisi(){
		return $this->posisi;
	}

}

?>